django/db/backends/oracle/introspection.py

```diff
@@ -8,6 +9,8 @@
 from django.utils.deprecation import RemovedInDjango21Warning
 from django.utils.encoding import force_text
 
+FieldInfo = namedtuple('FieldInfo', FieldInfo._fields + ('default',))
```

Comment:
If all bulit-in backends are adding 'default' now, might as well put it in the the base FieldInfo, I think.